api/v6/products.thrift

```diff
@@ -25,7 +25,8 @@ struct ProductConfiguration {
   2:          string             endpoint,
   3:          string             displayedName_b64,
   4:          string             description_b64,
-  5: optional DatabaseConnection connection
+  5: optional DatabaseConnection connection,
+  6:          i64                runLimit
```

Comment:
Most of the other structs have `,` after the last element so adding a new member does not make the previous line modified.